.github/CONTRIBUTING.md

```diff
@@ -161,7 +161,7 @@ If you are porting features/tools from other codebases, you must give them credi
 
 Regarding sprites & sounds, you must credit the artist and possibly the codebase. All /tg/station assets including icons and sound are under a [Creative Commons 3.0 BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/) unless otherwise indicated.
 
-Regarding sprites in particular, you still need to take your own screenshots of the sprites in-game on your Skyrat code in your PR body, not just re-use the screenshots provided in the original PR.
+Regarding sprites in particular, you still need to take your own screenshots of the sprites in-game on your Bubber-Station code in your PR body, not just re-use the screenshots provided in the original PR.
```

Comment:
Do we even want to require this from  contributors? CI & MDB exist for a reason.

Reply:
I spotted it and changed it. Not a giant deal.